hw/top_earlgrey/doc/design/README.md

```diff
@@ -540,6 +540,21 @@ The first EDN instance, `u_edn0` is intended to be configured to deliver entropy
 
 <!-- END CMDGEN -->
 
+#### Open drain / virtual open drain
+
+Pads of type **BidirOd** and **BidirStd** support **open drain** and **virtual open drain**, respectively.
+When open drain or virtual open drain is enabled in pinmux's attributes for such a pad, the PMOS transistor that would otherwise drive the output high for a logic 1 is disabled.
+Thus for a logic 1, the impedance of the pad is essentially the output impedance of the CMOS transistor pair (which is specified in the data sheet of the chip).
+
+However, there still exists a parallel connection to the supply rail through a protection circuit.
+Therefore:
+- For **BidirOd** pads in open drain mode, the output protection circuit allows voltages that exceed the supply voltage.
```

Comment:
The statement as written seems to imply that BidirOd pads not in open drain mode do not allow voltages that exceed the supply voltage.  Is that an accurate statement?

If so, I think there is a potential problem.  As far as I can tell the reset default for MIO_PAD_ATTR is zero for all pads, meaning that a random watchdog reset would put all open drain pads into a mode in which they do not tolerate voltages exceeding the supply.  If that is true, it would severely limit the usefulness of the open drain pads.  Do we need to modify the reset default for some pads to have open drain enabled, or do the BidirOd pads tolerate voltages in excess of the supply voltage under other circumstances, such as when "output enable" is not asserted?